Packages/com.unity.inputsystem/InputSystem/Editor/Settings/InputSettingsBuildProvider.cs

```diff
@@ -19,14 +19,12 @@ public void OnPreprocessBuild(BuildReport report)
             var preloadedAssets = PlayerSettings.GetPreloadedAssets();
 
 #if UNITY_INPUT_SYSTEM_PROJECT_WIDE_ACTIONS
-            EditorBuildSettings.TryGetConfigObject(InputSettingsProvider.kEditorBuildSettingsActionsConfigKey,
-                out m_ProjectWideActions);
-
+            m_ProjectWideActions = Editor.ProjectWideActionsAsset.GetOrCreate();
```

Comment:
Seems reasonable and was feedback in a previous PR.
I'm note sure why the config key was used previously though. Assuming no side effects this looks tidier.

Reply:
I added it recently based on the documentation for using preloaded assets, but on reflection it is unnecessary in our case as we still have access to `AssetDatabase` when this code is run.